tensorflow/compiler/xla/service/gpu/llvm_gpu_backend/gpu_backend_lib.cc

```diff
@@ -752,7 +752,8 @@ StatusOr<std::vector<uint8_t>> EmitModuleToHsaco(
   // Locate lld.
   // TODO(whchung@gmail.com): change to tensorflow::ROCmRoot() after
   // ROCm-Device-Libs PR.
-  std::string lld_path = tsl::io::JoinPath("/opt/rocm", "llvm/bin");
+  // add support for dcu
+  std::string lld_path = tsl::io::JoinPath("/opt/dtk-22.04.2", "llvm/bin");
```

Comment:
may guard with macro like `IF_DCU`? or read from env var ... not ok to hard code one.